ntcore/src/main/native/cpp/Value.cpp

```diff
@@ -317,6 +317,10 @@ bool nt::operator==(const Value& lhs, const Value& rhs) {
       if (lhs.m_val.data.arr_boolean.size != rhs.m_val.data.arr_boolean.size) {
         return false;
       }
+      if (lhs.m_val.data.arr_boolean.arr == nullptr &&
+          rhs.m_val.data.arr_boolean.arr == nullptr) {
```

Comment:
These should be || rather than && for the best defensive check.  We already checked that the sizes are the same.